docs/core/config/sieve/index.md

```diff
@@ -188,22 +189,30 @@ global scripts using the `sievec` command line tool. For example:
 sievec /var/lib/dovecot/sieve/global/
 ```
 
-This is necessary for scripts listed in [[setting,sieve_global]],
-[[setting,sieve_before]], and [[setting,sieve_after]] settings.
+This is necessary for script in storages with
+[[link,sieve_storage_type_after,after]],
+[[link,sieve_storage_type_before,before]],
+[[link,sieve_storage_type_default,default]], and
+[[link,sieve_storage_type_global,global]] storage type.
 
 For global scripts that are only included in other scripts using the Sieve
-include extension, this step is not necessary since included scripts
-are incorporated into the binary produced for the main script.
+include extension (from the `personal` and `global` storage types), this step is
```

Comment:
`[[link,sieve_storage_type_personal,personal]] and [[link,sieve_storage_type_global,global]] storage types`

Reply:
fixed